install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -86,3 +86,14 @@ Return the appropriate apiVersion for cronjob.
 {{- print "batch/v1beta1" -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return the appropriate apiVersion for podDisruptionBudget.
+*/}}
+{{- define "podDisruptionBudget.apiVersion" -}}
+{{- if semverCompare ">=1.24-0" .Capabilities.KubeVersion.Version -}}
```

Comment:
sorry that I didn't make it clear in my previous comment. 

PDB is stable from 1.21 as per https://kubernetes.io/docs/tasks/run-application/configure-pdb/, so this check here should be >=1.21.0 instead i.e. if the kube cluster is >=1.21, we should use policy/v1 only.

Reply:
thanks, have fixed.